src/content/blog/2023/email-reader-view.md

```diff
@@ -0,0 +1,50 @@
+
+This is inspired by web reader view features have become popular over the last few years, with built in versions in many browser as well as a number of browser extension. 
+
+## Why did we build it
+
+There are a number of goals we had with this project 
+
+### Help users with consuming content
+
+Accessibility is a big goal for us at the EMC. This tool could help a number of users consume email content more easily. If a user struggles to read a certain font, or if the text is too small to read, or if they experience glare or halo effects on text with certain color combinations these can all be fixed. We also have an option to replace images with their alt text so text in images can now be more readable (assuming the sender has added good quality alt text).
+
+Personally I use reader view on webpages a lot and since working on this project, I’ve started using it in email often too.
+
+### Help with email client rendering issues
+
+Sometimes emails can display badly due to various issue in different email clients or badly written code from the sender. Applying reader view removes all of the styling, simplifying the content and should mean the content is always readable despite these potential issues. A common issue is an email that isn’t built to be responsive can be scaled much better now using reader view. 
+
+If you find an example where an email is displaying poorly when using reader view, please let us know.
+
+### Encourage developers to use semantic markup
+
+We’re always trying to get email senders to use semantic code, this tool helps emphasise the importance of semantic structure and hierarchy in a very visual way. When viewed in reader view, well written semantic code, simply looks better. If developers write code to optimise for reader view, the benefits should also apply more widely to the accessibility of the email.
+
+Additionally using the block images option makes checking alt text much easier, allowing you to easily read it in the flow of the email content, much like how a screen reader would.
+
+### Encourage email clients to build their own
+
+We think this would be a very valuable feature to build directly into the email clients. We’re hoping this project will help encourage the email clients to follow suit and build their own. The code is all open source so if you are from an email client, please feel free to read over it.
+
+## What next
+
+In the future we are hoping to add support for more email clients and support for more browsers as well as looking at any feedback we get from users. But the EMC members behind this have day jobs as well as other commitments, so don’t expect a lot of updates.
```

Comment:
Not sure whether the last part is needed. Something like the following may sound better:

> This is a free and open-source project, and updates can be slow.